odl/test/solvers/nonsmooth/adupdates_test.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2014-2017 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Test for the Fessler alternating dual updates solver."""
```

Comment:
Since "Fessler" is not part of the ODL name, it should not be mentioned here either